src/common/Dropdown/DropdownComponent.tsx

```diff
@@ -200,7 +201,10 @@ export const Dropdown = forwardRef<DropdownRef, DropdownProps>(({
       name="dropdown"
       data-testid={props.dataTestId}
       mix={[props.className, visibilityClasses]}
-      style={compositeStyles}
+      style={{
+        ...compositeStyles,
+        borderRadius: isFF(FF_DEV_3873) && 4,
```

Comment:
```suggestion
        borderRadius: isFF(FF_DEV_3873) ? 4 : undefined,
```

Reply:
Not sure, but I would make sure this doesn't pollute the styles with `borderRadius: false`. If it doesn't, disregard this change. If it does, we should adjust for it so we aren't possibly creating problems when the flag is off.